src/sparseml/pytorch/utils/logger.py

```diff
@@ -204,6 +205,7 @@ def lambda_func(
             Optional[Dict[str, float]],
             Optional[int],
             Optional[float],
+            Optional[int],
```

Comment:
I expected to see some description about what "level" means in the comment below.